commons-test/src/main/java/org/infinispan/commons/lambda/NamedBiConsumer.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.commons.lambda;
+
+import java.util.function.BiConsumer;
+
+public class NamedBiConsumer<T, U> implements BiConsumer<T, U> {
```

Comment:
I know it's probably more trouble than it's worth, but I'd like `NamedBiConsumer` to be nested in a `NamedLambdas` class with a `NamedLambdas.of(BiConsumer)` static method, so it's easier to add more functional interfaces later.

Reply:
@danberindei let me know what do you think

Reply:
@diegolovison If you make `NamedBiConsumer` package-private or a non-public inner class of `NamedLambdas` we can guarantee that `NamedLambdas#of` will always be called. At the minute somebody writing a test could still do `new NamedBiConsumer`. Personally I prefer to have everything in `NamedLambdas`.

Reply:
@ryanemerson Done